### ISSUE TITLE
[Other - Accessibility] Unnecessary `nav` landmarks. (00.00.2)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#55275]
#### VFS team name
CTO 

#### VFS product name
MHV on VA.gov Portal

#### VFS feature name
Authenticated Landing Page

#### Point of Contact/Reviewers
Brian DeConinck - @briandeconinck - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
No VA.gov Experience Standard for the issue found.

### Issue Details
Screen reader users often jump between sections of the page via [landmarks](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/roles/landmark_role). Typically you use it to jump between major structural parts of the page that are reasonably consistent across sections of the website: header, footer, left navigation etc. Elsewhere on VA.gov, pages have roughly 7 to 10 landmarks (example: My VA has 8).

As currently coded, each list of links on the My HealtheVet landing page is wrapped in a `nav` element, which means there are 16 landmarks with those 9 sections. Moreover, those `nav` elements are unlabeled --- meaning a screen reader user will have to sift through 9 nav sections (each list of links) that don't provide any context about what they are when viewing the landmark list.

While not a WCAG violation, this is a degraded experience for screen reader users and not consistent with the rest of VA.gov.

### VA.gov Experience Standard
[Category Number 00, Issue Number 00](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Platform Recommendation
The `nav` elements around each list of links aren't necessary. Just an H2 heading followed by a `ul` list is sufficient structure.

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by dcloud** [comment]
It sounds like reducing the number of landmarks is preferred over providing `aria-labelledby` on those nav elements. Looking at [MDN for reference](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav#usage_notes).

*Just adding notes here, not seeking alternatives: will remove `nav`.*

**Comment by briandeconinck** [comment]
Correct -- but this is very much context-dependent. There's definitely a situation where `aria-labelledby` to label the navs would be the right move. But in this case, it's probably better to stay consistent with the rest of VA.gov and user expectations for how the page is structured.
